SDWebImage/SDWebImageImageIOCoder.m

```diff
@@ -469,6 +471,28 @@ + (BOOL)shouldDecodeImage:(nullable UIImage *)image {
     return YES;
 }
 
++ (BOOL)canEncodeToHEICFormat
```

Comment:
I think we consider too much. Easy and direct way to detect HEIC encoding compatibility 😆

Reply:
Should iOS version check be good enough?

Reply:
@skyline75489 NO. Check my description and Apple's WWDC introduction. It only works on (iOS 11 || macOS 10.13) && (isMac || isIPhoneAndA10FusionChipAbove) && (!Simulator). Since it's `Natively` support and not effect all the interface above Image/IO layer, so we do not check that. Instead, just use Image/IO Public API to detect.